src/components/vr_cooperation/include/vr_cooperation/commands/base_command_request.h

```diff
@@ -47,7 +48,7 @@ namespace commands {
  */
 class BaseCommandRequest : public Command,
   public event_engine::EventObserver<
-  application_manager::MessagePtr, std::string> {
+  application_manager::MessagePtr, const std::string&> {
```

Comment:
I'm not sure about using const& for string. Who creates and deletes these strings?